Cargo.toml

```diff
@@ -343,7 +339,7 @@ opt-level = 2
 
 [patch.crates-io]
 # Patch third-party crates for deterministic simulation.
-quanta = { git = "https://github.com/madsim-rs/quanta.git", rev = "948bdc3" }
+quanta = { git = "https://github.com/tabVersion/quanta.git", rev = "bb6c780894d06c0ec3f487d58c72920665b5cb0a" }
```

Comment:
We may contribute to `madsim-rs/quanta`.

Reply:
let's merge https://github.com/madsim-rs/quanta/pull/2 first and we can switch back to madsim.